apps/idos-data-dashboard/src/routes/dashboard/wallets/index.tsx

```diff
@@ -46,6 +48,12 @@ const WalletsList = () => {
   const wallets = useFetchWallets();
   const { isOpen, onOpen, onClose } = useDisclosure();
   const [walletsToDelete, setWalletToDelete] = useState<idOSWallet[] | undefined>([]);
+  const { address: ethAddress } = useAccount();
+  const { accounts } = useWalletSelector();
+
+  const nearAddress = accounts?.[0]?.accountId;
+
+  const address = ethAddress || nearAddress;
```

Comment:
```suggestion
  const connectedWalletAddress = ethAddress || nearAddress;
```

It took me a while to understand what was going on here. I think this name would have served me better.